composer.json

```diff
@@ -17,7 +17,6 @@
     },
     "require-dev": {
         "easyswoole/easyswoole": "~3.3|~3.4",
-        "easyswoole/ddl": "^1.0",
```

Comment:
@basakest why delete this?

Reply:
I am not familiar with EasySwoole. So I try to create a data table through ddl according to the document. But it seems that because I didn’t use it correctly, the data table was not created correctly. I don't know how to fix this problem. So I temporarily chose to create a data table through Github Action.

Reply:
@gaobinzhan

Reply:
I see. I'll take the time to adjust the code of this library. After all, I'm going to have `casbin` next.

Reply:
@gaobinzhan thanks! Looking forward to your code contribution to adjust this part.